src/OrchardCore.Modules/OrchardCore.Menu/Routing/ContentPickerItemRoute.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Options;
+using OrchardCore.ContentManagement.Routing;
+
+namespace OrchardCore.Menu.Routing
+{
+    public class ContentPickerItemRoute
+    {
```

Comment:
I think we can skip this service. Just do what you need to (it's 3 lines right?) in the view